src/components/global/footer.tsx

```diff
@@ -1,9 +1,263 @@
 import { EXAMPLE_PATH } from "../../lib/constants";
+import styled from "@emotion/styled";
+import { GridContainer } from "../global/grid/gridContainer";
+import { PillIconButton as OriginalPillIconButton } from "../global/pillIconButton";
+import Pill from "../global/pill";
+import { breakpoints, colors, dimensions } from "../../styles/variables";
+import ChatIcon from "../../icons/chatIcon";
+import chatIcon from "../../../public/svg/icon-chat.svg";
+import Image from "next/image";
+import Link from "next/link";
+import { getRemSize } from "../../styles/globalCss";
+import eoanPicture from "../../../public/eoan-picture.png";
+import {
+  StyledParagraphWrapper as OriginalStyledParagraphWrapper,
+  StyledParagraphText,
+  StyledPillWrapper,
+  StyledTextSpacer,
+} from "../../components/home/approach";
+import useIsDesktop from "../../hooks/useIsDesktop";
+
+const StyledWrapper = styled(GridContainer)`
+  margin: 300px 0 260px 0;
+`;
+
+const PillIconButton = styled(OriginalPillIconButton)`
+  && {
+  margin: 24px 0;
+  }
+`;
+
+const StyledContent = styled.div`
+  grid-column: 3 / span 8;
+  padding: 0;
+  margin: 0;
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    grid-column: 1 / span 12;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    grid-column: 1 / span 12;
+  }
+`;
+
+// Imported from approach.tsx, but with additional styles
+const StyledParagraphWrapper = styled(OriginalStyledParagraphWrapper)`
+  @media all and (max-width: ${breakpoints.md}px) {
+    padding: initial;
+    margin: initial;
+    display: initial;
+    flex-direction: initial;
+    box-sizing: initial;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    padding: initial;
+    margin: initial;
+    display: initial;
+    flex-direction: initial;
+    box-sizing: initial;
+  }
+`;
+
+const StyledImage = styled(Image)`
+  width: 44px;
+  height: 44px;
+  margin-left: 0.7rem;
+  vertical-align: middle;
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    display: none;
+  }
+`;
+
+const StyledIconButton = styled.button`
+  width: 44px;
+  height: 44px;
+  margin: 0 0 0 0.7rem;
+  padding: 0;
+  border: 2px solid ${colors.blackLight};
+  transition: 0.3s ease;
+  border-radius: 0.75rem;
+  vertical-align: middle;
+
+  &:hover {
+    background-color: ${colors.accent};
+    border-color: ${colors.accent};
+  }
+`;
+
+const StyledLinkWrapper = styled.div`
+  display: flex;
+  height: 9px;
+  color: #F8F8F8;
+  font-weight: 400;
+  opacity: 0.5;
+  margin-bottom: 17px;
+  justify-content: end;
+
+  & > a {
+    margin-right: 1.5rem;
+  }
+
+  @media all and (max-width: ${breakpoints.md}px) {
+    justify-content: center;
+  }
+
+  @media all and (max-width: ${breakpoints.sm}px) {
+    justify-content: center;
+    font-size: 0.8rem;
```

Comment:
Try to use the pre defined font-size variables where you can
```suggestion
    font-size: ${getRemSize(dimensions.textSizes.small.mobile)};
```

They are matched up with the Style Guide on Figma